CMakeLists.txt

```diff
@@ -291,6 +293,15 @@ if(SENTRY_TRANSPORT_CURL)
 	endif()
 endif()
 
+if(SENTRY_TRANSPORT_COMPRESSION)
+	if(NOT ZLIB_FOUND)
+		find_package(ZLIB REQUIRED)
+	endif()
+
```

Comment:
As suggested here: https://github.com/getsentry/sentry-native/pull/954#discussion_r1502525718, you will need to inform the crashpad build to use the system zlib, or else we'll have two different zlib versions linked.

Please add:

```
set(CRASHPAD_ZLIB_SYSTEM ON CACHE BOOL "Force CRASHPAD_ZLIB_SYSTEM when enabling transport compression" FORCE)
```

This will further require a change in the crashpad build where the `find_package(ZLIB REQUIRED)` will also have to wrapped with a check for `ZLIB_FOUND`.

Reply:
Thank you for your careful review and helpful suggestions. I fixed them.